lib/postcode_validation/gateway/pca_address_list.rb

```diff
@@ -0,0 +1,43 @@
+module PostcodeValidation
+  module Gateway
+    class PCAAddressList
+      include HTTParty
+
+      class PCARequestError < PostcodeValidation::Error::RequestError; end
+
+      KEY = ENV['POSTCODE_ANYWHERE_KEY']
+      base_uri 'https://services.postcodeanywhere.co.uk'
```

Comment:
These APIs behave differently. The new one could match on things like company name which could give false positives when validating the postcode - i.e. http://www.pcapredict.com/support/webservice/postcodeanywhere/interactive/find/1.1/